pymeasure/instruments/teledyne/teledyneMAUI.py

```
@@ -26,6 +26,193 @@
+        ris_sweeps          = unpack('<h',desc[322:324])[0]
+        time_base           = unpack('<h',desc[324:326])[0]
+        vcoupling           = unpack('<h',desc[326:328])[0]
+        probe_atten         = unpack('<f',desc[328:332])[0]
+        fixed_vgain         = unpack('<h',desc[332:334])[0]
+        bwlimit             = unpack('<h',desc[334:336])[0]
+        vertical_vernier    = unpack('<f',desc[336:340])[0]
+        acq_vos             = unpack('<f',desc[340:344])[0]
+        wave_src            = unpack('<h',desc[344:346])[0]
+
+        return cls(desc_name, desc_templ_name, desc_comm_type, desc_comm_order, wave_desc_len,
+                    user_txt_len, trig_time_arr, ris_time_arr, wave_arr_1, wave_arr_2, instr_name,
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)